turbo.json

```diff
@@ -3,6 +3,7 @@
   // All environment variables that should invalidate a cached build needs to be declared here
   "globalEnv": [
     "CI",
+    "GITHUB_ACTIONS",
```

Comment:
Adding this as `@sanity/pkg-utils` will need it in order to detect the GitHub CI and output GitHub Action specific markers: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-an-error-message